packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1114,7 +1127,6 @@ export class Table extends TableBase {
       },
       sseSpecification,
       streamSpecification,
-      timeToLiveSpecification: props.timeToLiveAttribute ? { attributeName: props.timeToLiveAttribute, enabled: true } : undefined,
```

Comment:
Hmm, this is interesting...

I'm actually surprised you're not setting this in the template anymore... I feel like that's a mistake?

Reply:
This was intended by me. As we handle changing the TTL with the custom resource, it also must provide the functionality of simply setting it. Leaving it out of the template keeps the setting and updating logic completely within the custom resource. Additionally, if I remember that correctly, the Time to Live in CFN is not verified against the actual status of the DynamoDB when doing a drift detection, so this will not get flagged. All in all, it appeared cleaner to me to outsource the time to live completely.